Makefile

```diff
@@ -3,8 +3,9 @@
 # Build native Truss by default.
 default: truss
 
-dependencies: 
-	go install github.com/golang/protobuf/protoc-gen-go
+dependencies:
+	go get -u google.golang.org/genproto
+	go get -u github.com/golang/protobuf/{proto,protoc-gen-go}
```

Comment:
If these are being added I believe `go-kit/kit` should be as well. As `protoc-gen-go` is the only truss dependency here. The rest `proto` and `genproto` are only dependencies of the services that truss generates.